src/main/java/com/stripe/model/Person.java

```diff
@@ -132,16 +144,123 @@ protected String getInstanceUrl() {
     List<String> pastDue;
   }
 
-  // TODO: Move this to a top level class instead of duplicating
-  // with the LegalEntity resource
   @Getter
   @Setter
   @EqualsAndHashCode(callSuper = false)
   public static class Verification extends StripeObject {
     String details;
     String detailsCode;
-    String document;
-    String documentBack;
+    VerificationDocument documentSubObject;
     String status;
+
+    /**
+     * The {@code document} attribute.
+     *
+     * @return the {@code document} attribute
+     * @deprecated Prefer using the {@link #getDocumentSubObject} method instead.
+     * @see <a href="https://stripe.com/docs/upgrades#2019-02-19">API version 2019-02-19</a>
+     */
+    @Deprecated
+    @Getter(AccessLevel.NONE) @Setter(AccessLevel.NONE) ExpandableField<File> document;
+
+    /**
+     * The {@code document_back} attribute.
+     *
+     * @return the {@code document_back} attribute
+     * @deprecated Prefer using the {@link #getDocumentSubObject} method instead.
+     * @see <a href="https://stripe.com/docs/upgrades#2019-02-19">API version 2019-02-19</a>
+     */
+    @Deprecated
+    @Getter(AccessLevel.NONE) @Setter(AccessLevel.NONE) ExpandableField<File> documentBack;
+
+    // <editor-fold desc="document">
+    @Deprecated
```

Comment:
We should copy the Javadoc comments for `document` on the `getDocument()` and `getDocumentObject()` getters, otherwise library users won't be able to see it. Same thing for `documentBack` -> `getDocumentBack()` + `getDocumentBackObject()`.

Reply:
@ob-stripe Fixed, can you confirm it's what you wanted?

Reply:
Yup, thanks!